drafter.gyp

```diff
@@ -111,6 +111,7 @@
       'type': 'executable',
       'include_dirs': [
         'src',
+        'src/refract',
```

Comment:
Yes, It wasn't working for a long time and I neglected it since I was able to work around it.

https://github.com/apiaryio/drafter/pull/265